.tekton/determine-image-tag-task.yaml

```diff
@@ -8,19 +8,19 @@ spec:
   description: Determines the tag for the output image using the StackRox convention from 'make tag' output.
   params:
   results:
-  - name: image-tag
-    description: Image Tag determined by custom logic.
+    - name: image-tag
+      description: Image Tag determined by custom logic.
   steps:
-  - name: determine-image-tag
-    image: registry.access.redhat.com/ubi8:latest
-    script: |
-      #!/usr/bin/env bash
-      set -euo pipefail
-      dnf -y upgrade --nobest
-      dnf -y install git make
-      cd "$(workspaces.source.path)/source"
-      scripts/konflux/fail-build-if-git-is-dirty.sh
-      echo -n "$(make --quiet --no-print-directory tag)-fast" | tee "$(results.image-tag.path)"
+    - name: determine-image-tag
```

Comment:
Could you do the same whitespace changes to `determine-image-tag` in `stackrox/stackrox` and `stackrox/collector` if not done already?

Reply:
Collector one will come in https://github.com/stackrox/collector/pull/1676
Will open a PR for Stackrox.

Reply:
@tommartensen the formatting thing turns to be slightly more elaborate than I thought.
Would you be ok if I merge this PR as-is and follow up with a set of new ones with formatting changes?

Reply:
Sure, go ahead